### ISSUE TITLE
Provide a CLI tool for even easier deployment

### ISSUE THREAD

**Issue by rashidnhm** [ONTechNITS/QuickHTML#21]
Instead of having the serve the app as part of your code, we can provide a CLI tool. This is common practice.

`main.py`:
```python
from quickhtml import QuickHtml
from quickhtml.tags import Html, H1

app = QuickHtml()

@app.route("/")
def index():
    return Html(H1("Hello world!"))
```

Now:
```bash
quickhtml serve main.py
```

And since you are using the CLI, this is considered dev and we automatically provide hotreloading.

**Comment by ThomasJRyan** [comment]
To add onto this, something I wish FastAPI and Flask had (maybe they do these days, not sure) is a command to quickly create a boilerplate project. Most JS frameworks have such a command so it'd be ideal to have one here as well

**Comment by rashidnhm** [comment]
I agree, for example, poetry gives you a `poetry init` command to quickly generate a pyproject.toml.

I see no reason why we can't provide the same!

**Comment by ThomasJRyan** [comment]
A sub command to convert an HTML file to QuickHTML would surely be useful
